docs/source/guides/converting-to-openapi.rst

```diff
@@ -468,7 +468,14 @@ service.
 CORS functionality
 ------------------
 
-TODO
+When the ``smithy.api#cors`` trait is applied to a service, Smithy performs the following
```

Comment:
This is a good start, but it needs to go a lot deeper into how this works. What does an example look like? How does it determine the headers to allow?